src/profile/components/Settings/Membership/Info.tsx

```diff
@@ -0,0 +1,14 @@
+import Markdown from 'common/components/Markdown';
+import React from 'react';
+
+const ABOUT_PROFILE_MEMBERSHIP = `
+  # Medlemskap
+
+  Du trenger medlemskap for å kunne melde deg på linje-spesifikke arrangementer.
+
+  For å søke medlemskap, bruk knappen under
+`;
```

Comment:
I don't see why this should be it's own component if the text is so tightly bound that it says "click the button below"